spec/support/acceptance/splunk/default.yml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
What's this file for?

Reply:
This file is passed into the container on startup and is used to configure all the splunk settings you'd normally configure via the web interface.

[https://splunk.github.io/docker-splunk/ADVANCED.html](https://splunk.github.io/docker-splunk/ADVANCED.html)

Reply:
Isn't this for configuring the splunk container?

Reply:
Ah ok. I'd maybe post a comment in the file along those lines. May be nice to link to a reference? E.g. something like https://splunk.github.io/docker-splunk/ADVANCED.html

Reply:
will do!